<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LauncherWindow</class>
 <widget class="QMainWindow" name="LauncherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LauncherWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(46, 52, 54);

QPushButton {
  background: rgb(186, 189, 182);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="newButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>440</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(186, 189, 182)</string>
    </property>
    <property name="text">
     <string>New Profile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="qBtn_start">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>240</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(186, 189, 182)</string>
    </property>
    <property name="text">
     <string>Start OpenSpace</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(186, 189, 182)</string>
    </property>
    <property name="text">
     <string>Edit Profile</string>
    </property>
   </widget>
   <widget class="QLabel" name="logolabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>400</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxProfiles">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>240</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxWindowConfigs">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>240</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelChoose">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ffffff;&quot;&gt;Choose Profile&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelOptions">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ffffff;&quot;&gt;Window Options&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSim">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>500</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>simData</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>473</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpenSpace_Launcher">
    <property name="title">
     <string>OpenSpace Launcher</string>
    </property>
   </widget>
   <addaction name="menuOpenSpace_Launcher"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
